client/src/components/shipping_info/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import "./style.css";
+import swal from "sweetalert";
+import NavBar from "../Navbar";
+import axios from "axios";
+
+class shippingInfo extends Component {
+  state = {
+    username: "",
+    phone: "",
+    Address: "",
+    Extra_Note: ""
+  };
+
+  changeInput = ({ target: { value, name } }) => {
+    this.setState({ [name]: value });
+  };
+
+  clickButton = () => {
+    const { username, phone, Address, Extra_Note } = this.state;
+    if (username !== "" && phone !== "" && Address !== "") {
+      axios
+        .post("/shipping_info", { username, phone, Address, Extra_Note })
+        .then(res => console.log("5555555555", res.data));
```

Comment:
what is `5555555555` mean in `.then(res => console.log("5555555555", res.data));`

Reply:
deleted it , just for test . @ishak52